src/lib/api.js

```diff
@@ -388,20 +390,34 @@ const searchItems = async function (collectionId, queryParameters, backend, endp
   }
 
   // Keep only existing parameters
-  const searchParameters = Object.keys(parameters)
-    .filter((key) => parameters[key])
-    .reduce((obj, key) => ({
-      ...obj,
-      [key]: parameters[key]
-    }), {})
+  const searchParameters = pickBy(parameters)
```

Comment:
are parameters that have the value `0` going to be strings here and therefore truthy?

Reply:
The behavior will be the same as it was before, where the value's truthiness is what's evaluated. That being said, we really should start doing _some_ type checking. I've found a number of places where the behavior is going to be suspect.

Reply:
agree. The last STAC API worked on was in Scala, and about half the code and 3/4 of our time were on validation and error handling.